robolectric-utils/src/main/java/org/robolectric/internal/bytecode/InvokeDynamic.java

```diff
@@ -0,0 +1,21 @@
+package org.robolectric.internal.bytecode;
+
+public class InvokeDynamic {
+  public static final boolean ENABLED = useInvokeDynamic();
+
+  private static final String ENABLE_INVOKEDYNAMIC = "robolectric.invokedynamic.enable";
```

Comment:
That is a good question. Honestly I think the default is fine and as invokedynamic is a pure win there shouldn't be any need to disable it so it can probably be undocumented for now. Probably worth to mention in any release notes however.